ansible/ansible.cfg

```diff
@@ -9,9 +9,24 @@
 # Set `ANSIBLE_CONFIG` for local changes
 
 [defaults]
+# Try more servers at once, smartly.
+forks = 20
+gathering = smart
+
+# Assume all actions take place under infrastructure/ansible
 roles_path = roles/
+inventory = ../../ansible/inventory/
+
+# These tend to be annoying.
 retry_files_enabled = False
 
-# Allow becoming an unprivileged user
-# http://docs.ansible.com/ansible/become.html#becoming-an-unprivileged-user
-allow_world_readable_tmpfiles = True
+# Deals with the files/ directory used by IDR
+inventory_ignore_extensions = ~, .orig, .bak, .ini, .retry, .pyc, .pyo, .html, .omero, .gif, .png, .js, .md
+
+# Helps determine what's running slowly
+callback_whitelist = profile_tasks
+
+# Performance options
+[ssh_connection]
+ssh_args = -o ForwardAgent=yes -o ControlMaster=auto -o ControlPersist=60s
```

Comment:
I'm drawing something of a blank, but I think the reason for adding this was for getting through the gateways. It might be that we want to add configuration at the host/group level for anything behind a gateway. In general, you and I should talk through the public/private split, since I'm sure I've left things in a surprising state for you.